module2.js

```diff
@@ -22,7 +22,11 @@
 
 function question1(personObj) {
   // TODO
-  return /* ??? */;
+  return `Hi, my name is ${personObj.firstName} ${personObj.lastName}. I am ${
+    personObj.age
+  } years old and work as a
+  ${personObj.occupation} in ${personObj.location}.`;
```

Comment:
Try and run the tests before submitting - this one doesn't pass. Only because of the line break between lines 27 and 28 though :)